### PR TITLE
Major configuration, documentation clean-up

### PR DESCRIPTION
**Pull request by nonword**
This work:
 - Fixes confusing configuration, previously split between `config`
   module and dot-env variables; Now using dot-env var exclusively.
 - Better README and inline documentation in jobs/update-(bibs|items).js
   scripts
 - Remove deprecated "Controlled Vocab" serialization (now using
   nypl-core directly)
 - Removes shadowcat serialization completely as the service has been
   deprecated
 - Ensure kinesify-data uses new official local config
 - DB entity caching now controlled by env var, default false
 - Remove `config` from package.json
 - Make nypl-core version env-configurable
 - Update README to reflect qa deploy branch

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,126 +12,126 @@ Before one can run or deploy, one needs to do two things:
...
+`node jobs/init check --envfile [path to env file] --profile [aws profile]`
 
 ### Creating a Test Event
 
-Run this to generate `event.bibs.json` full of encoded bibs:
+Test event json files can be generated to emulate lambda invocations locally.
+
+Run this to generate `event.json` full of encoded bibs:
 
 `node kinesify-data.js event.unencoded.bibs.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Bib`
```
Comment (holingpoon) [inline README.md]: Would like to see flags included in README, though seeing usage suggestions on the command line is very helpful.

`node kinesify-data.js event.unencoded.bibs.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Bib  --profile [aws profile] --envfile [path to env file]`

File: README.md [hunk 2]
```diff
@@ -12,126 +12,126 @@ Before one can run or deploy, one needs to do two things:
...
 
-Or run this to generate `event.items.json` full of encoded items:
+Or run this to generate `event.json` full of encoded items:
 
 `node kinesify-data.js event.unencoded.items.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Item`
```
Comment (holingpoon) [inline README.md]: Would like to see flags included in README, thought seeing usage suggestions on the command line is very helpful.
`node kinesify-data.js event.unencoded.items.json event.json https://platform.nypl.org/api/v0.1/current-schemas/Item  --profile [aws profile] --envfile [path to env file]`

File: README.md [hunk 3]
```diff
@@ -12,126 +12,126 @@ Before one can run or deploy, one needs to do two things:
...
 
 Alternatively, to generate a event.json from a plain marcinjson document (such as retrieved from the sierra api), you can kinesify that directly:
 
 `node kinesify-data test/data/bib-10011745.json event.json  https://platform.nypl.org/api/v0.1/current-schemas/Bib`
```
Comment (holingpoon) [inline README.md]: Would like to see flags included in README, thought seeing usage suggestions on the command line is very helpful.
`node kinesify-data test/data/bib-10011745.json event.json  https://platform.nypl.org/api/v0.1/current-schemas/Bib --profile [aws profile] --envfile [path to env file]`

File: README.md (hunk 1 above)
Comment (nonword) [reply README.md]: Good catch. Added.

**Pull request merged**